src/editconfig.cpp

```diff
@@ -0,0 +1,560 @@
+#include "editconfig.h"
```

Comment:
so many comments...  would betted be less imo

Reply:
Can you specify what's being overdocumented here? I wanna know what I can keep and toss out.